File: 10865.py
import sys

N, M = map(int, sys.stdin.readline().split())
friends = [0] * (N + 1)

for i in range(M):
    A, B = map(int, sys.stdin.readline().split())

    friends[A] += 1
    friends[B] += 1

for friend in friends[1:]:
    print(friend)

File: 10865.cpp
#include <iostream>
using namespace std;

int friends[100001];

int main() {
	int N, M; cin >> N >> M;

	int A, B;
	for (int i = 0; i < M; i++) {
		cin >> A >> B;

		friends[A]++;
		friends[B]++;
	}

	for (int i = 1; i <= N; i++)
		cout << friends[i] << '\n';
}
